_readings/sister-outsider.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: sister-outsider
+title: "Sister Outsider"
+authors: "Audre Lorde"
+categories: race
+goodreads_url: https://www.goodreads.com/book/show/32951.Sister_Outsider
+amazon_url: https://www.amazon.com/Sister-Outsider-Speeches-Crossing-Feminist-ebook/dp/B006L7RCEI/
```

Comment:
re: affiliate urls, it's an Amazon program that will give you commissions through purchases if visitors buy using the url. we can also pair it with [Amazon Smile](https://affiliate-program.amazon.com/help/node/topic/202049200) to earn them as donations to DU or have those profits go to another non-profit. We could probably think this out a little more and set it up in a different PR.

re: hyphens v underscores, in YAML the convention seems to be underscores in key names. values can have either, but if the values are used as a css value or a url slug, they should be hyphens. [This documentation](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html) doesn't clearly state it, but it seems to be the convention they follow.